engine/crates/engine-v2/src/utils.rs

```diff
@@ -24,6 +24,12 @@ macro_rules! id_newtypes {
                 }
             }
 
+            impl std::fmt::Debug for $name {
+                fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                    usize::from(*self).fmt(f)
```

Comment:
In debug output, it felt unnecessary to have something like
```
BoundField(
  plan_id = PlanId(
     0
  )
)
```
rather than just
```
BoundField(
  plan_id = 0
)
```

Given that nothing is dynamic here, it felt more verbose than anything IMHO

Reply:
Hmm...

Reply:
well haven't had any case showing up yet for me 🤷